src/components/ui/Button/buttons.scss

```diff
@@ -0,0 +1,66 @@
+@import "../../../styles/utilities";
+
+.button[data-store-button] {
+  display: flex;
```

Comment:
`button` elements are inline-block by default, so should this also be `inline-flex` as not to occupy the whole width?

`flex`|`inline-flex`
-|-
![Screen Shot 2021-12-28 at 21 39 15](https://user-images.githubusercontent.com/381395/147617102-627d3b42-f58a-4d48-9374-b65d2ecffa08.png)|![Screen Shot 2021-12-28 at 21 39 25](https://user-images.githubusercontent.com/381395/147617105-31b1084a-7505-4c79-adb8-a40f3fab710f.png)

Reply:
hmm, I think you are right!